lib/api/model/events.dart

```diff
@@ -900,16 +901,16 @@ class TypingEvent extends Event {
     required this.recipientIds,
     required this.streamId,
     required this.topic,
-  });
+  }) : assert(isSortedWithoutDuplicates(recipientIds ?? []));
```

Comment:
nit:
```suggestion
  }) : assert(recipientIds == null || isSortedWithoutDuplicates(recipientIds));
```

I think that's a bit clearer, because in the situation where `recipientIds` is null it's not that that's a way of expressing that the list of recipients is empty — rather it means it doesn't make sense to talk about recipients (because it's a stream/channel message).